internal/pkg/grpc/factory/factory.go

```diff
@@ -105,7 +106,15 @@ func NewListener(setters ...Option) (net.Listener, error) {
 	case "unix":
 		address = opts.SocketPath
 
-		// make any dirs on the path to the listening socket
+		// Unlink the address or we will get the error:
+		// bind: address already in use.
+		if _, err := os.Stat(address); err == nil {
+			if err := unix.Unlink(address); err != nil {
```

Comment:
didn't mean to be nitpicking here, but there's also `os.Remove` which doesn't require x/sys interface

Reply:
Good point. I can fix to use `os.Remove`.